database/generateData.js

```diff
@@ -1,55 +1,65 @@
 const fs = require('fs');
 const faker = require('faker');
 
-const write = function(filename, numRecords) {
-  return new Promise((resolve, reject) => {
+let name;
+let numRec;
+let i = 1;
+let stream;
+let callback;
```

Comment:
I don't believe these let variables (excluding i) are doing anything currently, as they are undefined. Did you mean to use them as default variables below in lines 12 ~ 15?